todo-backend/todo-backend.yaml

```diff
@@ -0,0 +1,43 @@
+# This configuration file can be used to build and deploy the todo-backend
+# quickstart on OpenShift with the Helm Chart for WildFly.
+build:
+  uri: https://github.com/jmesnil/quickstart.git
+  ref: todo-backend
+  mode: bootable-jar
+  env:
+    - name: ARTIFACT_DIR
+      value: todo-backend/target
+    - name: MAVEN_ARGS_APPEND
+      value: -am -pl todo-backend -P bootable-jar-openshift -Dversion.server.bom=23.0.0.Final -Dversion.microprofile.bom=23.0.0.Final
```

Comment:
Another comment:
> The backend is packaged as a Bootable Jar and configured to be deployable on OpenShift with the Maven Profile `bootable-jar-openshift` which is active
by default:

so there should be no need to specify it here (unless OpenShift adds something more which would disable the `active by default` feature

Reply:
fixed, thanks